content/java/concepts/set/terms/treeset/treeset.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Java-TreeSet' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Add treeSet implementation: ex. Set set = new TreeSet<>()' # Required; ideally under 150 characters and starts with a present-tense verb (used in search engine results and content previews)
```

Comment:
The description needs to be about `treeset` in java. Please make that change.

Reply:
I made this change. Please check.